scripts/gcs_bulk_delete.py

```diff
@@ -0,0 +1,144 @@
+import re
+import sys
+import time
+from datetime import datetime
+
+import google.auth
+from google.api_core import operations_v1
+from google.cloud import storage_transfer_v1
+from google.type.date_pb2 import Date
+from google.type.timeofday_pb2 import TimeOfDay
+
+
+EMPTY_BUCKET = "levanter-empty"
+
+
+def schedule_gcs_deletion_job(project_id, gcs_bucket_name, path_to_delete):
+    """Schedules an STS job to delete all files in a GCS path and waits for completion."""
+
+    client = storage_transfer_v1.StorageTransferServiceClient()
+
+    # Define the transfer job
+    transfer_job = storage_transfer_v1.types.TransferJob(
```

Comment:
Confused - is this a transfer job or a deletion job?

Reply:
I think it's a transfer job to an empty bucket

Reply:
Actually, it's a transfer from the Empty Bucker to the destination bucket, making sure `delete_objects_unique_in_sink` is true, which means "Whether objects that exist only in the sink should be deleted. "

Reply:
yes, it's a dumb trick for STS. I'll add a comment